identityClient/identityClient.go

```diff
@@ -679,7 +679,7 @@ func (c *E3dbIdentityClient) RealmInfo(ctx context.Context, realmName string) (*
 // PrivateRealmInfo fetches the private realm information based on realm name.
 func (c *E3dbIdentityClient) PrivateRealmInfo(ctx context.Context, realmName string) (*PrivateRealmInfo, error) {
 	var info *PrivateRealmInfo
-	path := c.Host + identityServiceBasePath + "/realm/info/" + realmName
+	path := c.Host + identityServiceBasePath + "/realm/" + realmName + "/info"
```

Comment:
you can't do this because cyclops doesn't support plugins on dynamic paths (in this case where realm name will be different each request), it has to be a static route prefix like realm/info